### ISSUE TITLE
Status --live sends unwanted script status messages via Telegram

### ISSUE THREAD

**Issue by zappra** [CoinAlpha/hummingbot#3077]
**Describe the bug**
Status --live will send unwanted script status messages via Telegram

**Steps To Reproduce**
Configure Telegram 
Set script enabled to true using script "inv_skew_using_spread_script.py"
Start any PMM strategy
Enter command "status --live"

Note that the script status message is sent every tick via Telegram

**Release version**
dev-0.37.0

**Comment by rapcmia** [comment]
We tried to replicate this on dev-0.38.0 and confirmed this is the same error when running `status --live` sending script status on telegram:

Telegram
![image](https://user-images.githubusercontent.com/73840223/111159902-8cfec580-85d4-11eb-94be-309566bde094.png)

Hummingbot dev-0.38.0
![image](https://user-images.githubusercontent.com/73840223/111160078-b881b000-85d4-11eb-9a05-a6723a74659e.png)

```
2021-03-15 21:13:23,348 - 8292 - hummingbot.script.script_iterator - INFO - received: CallNotify {'msg': 'Script status: \noriginal bid spread = 1.00% \noriginal ask spread = 1.00% \nask skew active? = False \ncurrent ask spread = 1.00% \nbid skew active? = False \ncurrent bid spread = 1.00%'}
```

Currently, our telegram integration is a bit of a hack and our team has plans to improve the integration in the future. When we use `status` bot would normally post an update on telegram however when this is `status --live` then it is expected to post every tick. We suggest for now disabling telegram when using the `status --live` command

**Comment by dennisocana** [comment]
Fixed in version 0.40 release by @zappra. Thanks for contributing!
